check_labels.py

```diff
@@ -174,14 +174,22 @@ def parse_condition(condition):
     return [e.strip() for e in elems]
 
 
-def check_review_approvals(pull, condition):
+def check_review_approvals(pull, condition, missing_approvals_label):
     condition[1] = int(condition[1])
     approvals = 0
     for review in pull.get_reviews():
         if review.state == "APPROVED":
             approvals += 1
             if approvals > condition[1]:
+                print(f"PR#{pull} has {approvals}/{condition[1] + 1} approvals, removing label \"{missing_approvals_label}\"")
```

Comment:
Existing prints use `'` within the string literals for quoting. What speaks against it here?
```suggestion
                print(f"PR#{pull} has {approvals}/{condition[1] + 1} approvals, removing label '{missing_approvals_label}'")
```